src/PrestaShopBundle/Resources/config/services/core/form/command_builder.yml

```diff
@@ -150,7 +150,10 @@ services:
     arguments:
       - !php/const PrestaShopBundle\Form\Admin\Extension\ModifyAllShopsExtension::MODIFY_ALL_SHOPS_PREFIX
 
-  prestashop.core.form.command_builder.product.combination.update_combination_commands_builder:
-    class: PrestaShop\PrestaShop\Core\Form\IdentifiableObject\CommandBuilder\Product\Combination\UpdateCombinationCommandsBuilder
+  PrestaShop\PrestaShop\Core\Form\IdentifiableObject\CommandBuilder\Product\Combination\UpdateCombinationCommandsBuilder: ~
```

Comment:
Need autowire true?

Reply:
I don't think so since it has no arguments, if some are added the autowire can be added when needed then